backend/tournaments/views.py

```diff
@@ -1,3 +1,65 @@
-from django.shortcuts import render
+from . import models
+from django.http import HttpResponse
+from django.shortcuts import get_object_or_404
+from django.core.exceptions import PermissionDenied
+import json
+import datetime
 
-# Create your views here.
+def get_participants(request, tournament):
```

Comment:
I'm going to prettify those functions a bit.
Will separate http response code from actual logic, so that those functions can also be called from a bot.
(in that case both bot and http would ideally not need to touch models directly at all).

You may want to wait with the review until I do that, or I can do that in a follow-up change.